osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/DeviceChannelsHelper.java

```diff
@@ -110,8 +114,8 @@ protected ChannelElementValuesDto makeChannelElementValues(final short channel,
     private String readIdentificationNumber(final List<GetResult> resultList, final int index, final String description)
```

Comment:
it seems strange to pass both the entire list and the index, and then get the item from the List based on that index. Why not pass only the item if you know the index beforehand.

Reply:
This is indeed weird. I just kept it the same as it was. If I change it, I should change more similar calls in this class. And I found that a too much for the small change I did.